core/src/script/CGXP/tools/autoProject.js

```diff
@@ -0,0 +1,91 @@
+/**
+* Copyright (c) 2011-2014 by Camptocamp SA
+*
+* CGXP is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* CGXP is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with CGXP. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/**
+* @requires OpenLayers/BaseTypes/Class.js
+* @requires OpenLayers/Util.js
+* @include OpenLayers/Map.js
+*/
+
+OpenLayers.AutoProjection = OpenLayers.Class({
+    initialize: function(config) {
+        this.buildProjectionList(config);
+        if (this.projections.length > 1){
+           this.isEnabled = true;
+        } else { 
+           this.isEnabled = false;
+        }
+    },
+
+    /** private: property[projections]
+     *  List of OpenLayers.Projection object
+     *  used to reproject some coordinates. It's based
+     *  on projectionCodes
```

Comment:
mh... projectionCodes is a parameter from the config. buildProjectionList transform it to "projections". I'm not sure of your remark...